docs/src/reference/roadmap.md

```diff
@@ -0,0 +1,15 @@
+# Roadmap
+
+The Canonical Kubernetes team recognises the value of having a public roadmap,
```

Comment:
Could we say something like "enthusiastically supports the idea of a public roadmap" instead of a neutral acknowledgement?

Reply:
sure

Reply:
I will be counting on you to enthusiastically keep it up to date :)